### DISCUSSION TITLE
request the refund 请求退款

### DISCUSSION THREAD

**Discussion by Abel-Wu2024** [getlantern/lantern-forum-cn#1083] [Q&A]
I'm traveling in Russia on business and the Lantern I purchased often fails to connect, which affects the use. 
I would like to request the refund.
[我在Russia出差，购买的Lantern经常连不上，影响使用。
申请退款。
![Receipt](https://github.com/user-attachments/assets/69b52807-38cb-42f8-a0e4-041ddcd3c42a)
![Request](https://github.com/user-attachments/assets/d6ab9765-2052-47e1-8273-a1695a8799cc)

**Comment by MaxMKMax** [comment]
@Abel-Wu2024 您好. 看来您已经提交好退款了. 过几天客服应该会处理了.

**Reply by Abel-Wu2024** [reply]
请问今天中午试着连上后，显示为免费版了，是不是意味着成功办理退款了？
麻烦帮我查下。因为我没有收到退款受理成功相关的任何邮件和通知。
如果，退款确实成功受理，那大概会在多长时间内收到退款金额呢？
谢谢您！
![image](https://github.com/user-attachments/assets/7f56df58-d17c-442f-9ebd-7b59711a6130)
![image](https://github.com/user-attachments/assets/a71ac3fc-0513-4276-9199-9f044791d7b5)

**Comment by MaxMKMax** [comment]
@Abel-Wu2024 我这里无法帮您查看. 但看起来退款成功了所以显示为免费版. 一般要 2-3 周才能搞定. 您也查看邮件客服有没有给您任何答复吗.
